django_manifeststaticfiles_enhanced/jslex.py

```
@@ -452,6 +452,10 @@ def find_import_export_strings(file_contents):
     matches = []
 
     def _append_match(token_tuple):
+        # we can't support template strings with variables
+        if token_tuple[1].startswith("`") and "${" in token_tuple[1]:
+            return
```

I'm in two minds, I can make a case that this should raise an exception. Be more consistent with the current approach to unprocessable files, and the person probably want's to know that this js file doesn't have cache busting due to the way the code is constructed.